app/views/forms/make_live/confirmation.html.erb

```diff
@@ -0,0 +1,23 @@
+<div class="govuk-grid-row">
```

Comment:
Should this file and `new.html.erb` be localised? I think we're still using Rails i18n everywhere else